lib/Job/Status.php

```diff
@@ -97,7 +97,8 @@ public function isTracking()
 	/**
 	 * Update the status indicator for the current job with a new status.
 	 *
-	 * @param int The status of the job (see constants in Resque\Job\Status)
+	 * @param int 	   $status The status of the job (see constants in Resque\Job\Status)
+	 * @param int|null $result The result of the job
```

Comment:
has to be `mixed`. `$result` can be pretty much anything returned from a Job's `perform()` method

Reply:
Anything JSONable, so technically `mixed` minus `reference`, but that's complicated to write with PHP type hints.